examples/with-amp/pages/_document.js

```diff
@@ -0,0 +1,29 @@
+import Document, { Head } from 'next/document'
+import { DOMProperty } from 'react-dom/../lib/ReactInjection'
+
+DOMProperty.injectDOMPropertyConfig({
```

Comment:
By default react limit the set of valid DOM elements and attributes https://github.com/facebook/react/issues/140 this config whitelist amp elements/attributes